src/sentry/tasks/post_process.py

```diff
@@ -213,6 +213,18 @@ def update_existing_attachments(event):
     )
 
 
+def fetch_buffered_group_stats(group):
+    """
+    Fetches buffered increments to `times_seen` for this group and adds them to the current
+    `times_seen`.
+    """
+    from sentry import buffer
+    from sentry.models import Group
+
+    result = buffer.get(Group, ["times_seen"], {"pk": group.id})
+    group.times_seen += result["times_seen"]
```

Comment:
This could be a footgun for future us if post_process ever ends up saving the group. Could we do something like have `group.pending` dict that had buffered changes?